pf/tfbridge/provider_create.go

```diff
@@ -44,6 +45,23 @@ func (p *provider) CreateWithContext(
 
 	priorState := newResourceState(ctx, &rh)
 
+	for key, fld := range rh.pulumiResourceInfo.Fields {
```

Comment:
We might want to abstract this into a func as we'll need to use it again.

Reply:
Introduced `SetDefaultValues`. Not sure about the method signature. Perhaps it's too narrow.